assets/scss/styles.scss

```diff
@@ -126,69 +152,52 @@ h1 {
 }
 
 /* Stats page */
-#stats {
+.stat_data {
+  h2 {
+    @include bold-24;
+  }
   table {
+    margin: 0 50px 50px;
     border-collapse: collapse;
     border-spacing: 0;
-    width: 100%;
-    margin-bottom: 50px;
+    box-sizing: border-box;
     thead {
       tr {
         th,
         td {
-          padding-top: 0;
-          padding-bottom: 16px;
+          padding: 0 1.25em 16px 0;
```

Comment:
Tom and I agree that using px based measurements is not accessible. This was based on Karwai's designs at the time. Will be revising.